### PR TITLE
feature/usd-liquidity-index: Adds USD Liquidity Index to Economy menu

### PR DESCRIPTION
**Pull request by deeleeramone**
# Description

- [X] Summary of the change / bug fix.

Frequently requested item, USD Liquidity Index.  Updated weekly and published to FRED, it is: WALCL - WLRRAL - WDTGAL

`/economy/usdli`
`openbb.economy.usdli`
`openbb.economy.usdli_chart`

- [X] Screenshot of the feature or the bug before/after fix, if applicable.

https://github.com/OpenBB-finance/OpenBBTerminal/assets/85772166/00a19a5f-6bb7-45e1-a53f-d36e2614b09c

```console
In [1]: from openbb_terminal.sdk import openbb

In [2]: openbb.economy.usdli()
Out[2]: 
            USD Liquidity Index (Billions of USD)  S&P 500 Index
2013-06-26                                3295.71        1603.26
2013-07-03                                3294.82        1615.41
2013-07-10                                3342.94        1652.62
2013-07-17                                3382.18        1680.91
2013-07-24                                3438.05        1685.94
...                                           ...            ...
2023-05-24                                5775.78        4115.24
2023-05-31                                5721.66        4179.83
2023-06-07                                5803.75        4267.52
2023-06-14                                5816.40        4372.59
2023-06-21                                5699.68        4365.69

[516 rows x 2 columns]
```

Screenshot 2023-06-22 at 10 29 42 PM

# How has this been tested?

* Please describe the tests that you ran to verify your changes.

The changes rely on existing FRED functions to pull data by Series ID.

- Added to integration test for the economy menu.
- Updated `test_economy_controller`
- Added tests to `tests/openbb_terminal/economy/test_fred_model.py`

# Checklist:

- [X] I have adhered to the GitFlow naming convention and my branch name is in the format of `feature/feature-name` or `hotfix/hotfix-name`.

- [X] Update [our documentation](https://openbb-finance.github.io/OpenBBTerminal/) following [these guidelines]
- [X] Update our tests following [these guidelines](https://github.com/OpenBB-finance/OpenBBTerminal/tree/main/tests).
- [X] Make sure you are following our [CONTRIBUTING guidelines](https://github.com/OpenBB-finance/OpenBBTerminal/blob/main/CONTRIBUTING.md).
- [X] If a feature was added make sure to add it to the corresponding [integration test script](https://github.com/OpenBB-finance/OpenBBTerminal/tree/develop/openbb_terminal/miscellaneous/integration_tests_scripts).

# Others
- [X] I have performed a self-review of my own code.
- [X] I have commented my code, particularly in hard-to-understand areas.

### CODE REVIEW

File: openbb_terminal/economy/fred_model.py [hunk 1]
```diff
@@ -348,3 +384,57 @@ def get_cpi(
     df = df.dropna()
 
     return df
+
+
+@log_start_end(log=logger)
+@check_api_key(["API_FRED_KEY"])
+def get_usd_liquidity(overlay: str = "SP500", show: bool = False) -> pd.DataFrame:
```
Comment (jmaslek) [inline openbb_terminal/economy/fred_model.py:391]: default the overlay to none so that by default, this only gets the calculated series

**Comment by jmaslek** [comment]
Should also be added to askobb.  @martinb-bb can provide guidance if needed

**Comment by martinb-bb** [comment]
@jmaslek  done
image

File: openbb_terminal/economy/fred_model.py (hunk 1 above)
Comment (deeleeramone) [reply openbb_terminal/economy/fred_model.py]: Done.

Screenshot 2023-07-05 at 11 34 22 AM

**Comment by jmaslek** [comment]
Can you ask @tehcoderer how to get rid of both major gridlines:

Screenshot 2023-07-05 at 5 16 32 PM

**Comment by deeleeramone** [comment]
> Can you ask @tehcoderer how to get rid of both major gridlines:

easy breezy. do you want the X-Grid gone too?

Screenshot 2023-07-05 at 2 21 53 PM

**Comment by jmaslek** [comment]
> > Can you ask @tehcoderer how to get rid of both major gridlines:
> 
> easy breezy. do you want the X-Grid gone too?
> 
> Screenshot 2023-07-05 at 2 21 53 PM

I like the major grid, but only for the main (right) axis.  Not too overlapped one

**Comment by deeleeramone** [comment]
> I like the major grid, but only for the main (right) axis. Not too overlapped one

So like this?

Screenshot 2023-07-05 at 2 29 01 PM

Would be pretty cool to have an on-chart setting for grid on/off.

**Comment by deeleeramone** [comment]
Personally I like it less busy, with just the horizontals.

Screenshot 2023-07-05 at 2 31 36 PM

**Comment by jmaslek** [comment]
> > I like the major grid, but only for the main (right) axis. Not too overlapped one
> 
> 
> 
> So like this?
> 
> 
> 
> Screenshot 2023-07-05 at 2 29 01 PM
> 
> 
> 
> Would be pretty cool to have an on-chart setting for grid on/off.

Dis one

**Comment by tehcoderer** [comment]
SHIP IT

**Review by jmaslek — APPROVED** [review]

**Pull request merged**